Classes/UIScrollView+InfiniteScroll.h

```diff
@@ -71,15 +81,15 @@ NS_ASSUME_NONNULL_BEGIN
  *
  *  @param handler a completion block handler called when animation finished
  */
-- (void)finishInfiniteScrollWithCompletion:(nullable void(^)(__pb_kindof(UIScrollView *) scrollView))handler;
+- (void)finishInfiniteScroll:(BOOL)animated completion:(nullable void(^)(__pb_kindof(UIScrollView *) scrollView))handler;
 
 /**
  *  Finish infinite scroll animations
  *
  *  You must call this method from your infinite scroll handler to finish all
  *  animations properly and reset infinite scroll state
  */
-- (void)finishInfiniteScroll;
+- (void)finishInfiniteScroll:(BOOL)animated;
```

Comment:
Would be better to add DEPRECATED attribute and keep the method around for a while so others could adjust and start using new methods.